cli/config/compose/profiles/fleet/docker-compose.yml

```diff
@@ -17,30 +17,23 @@ services:
       - ELASTIC_USERNAME=elastic
       - ELASTIC_PASSWORD=changeme
     image: "docker.elastic.co/elasticsearch/elasticsearch:${stackVersion:-8.0.0-SNAPSHOT}"
+    platform: ${stackPlatform:-linux/amd64}
     ports:
       - "9200:9200"
   kibana:
     depends_on:
       elasticsearch:
         condition: service_healthy
-      package-registry:
-        condition: service_healthy
     healthcheck:
       test: "curl -f http://localhost:5601/login | grep kbn-injected-metadata 2>&1 >/dev/null"
       retries: 600
       interval: 1s
     image: "docker.elastic.co/${kibanaDockerNamespace:-kibana}/kibana:${kibanaVersion:-8.0.0-SNAPSHOT}"
+    platform: ${stackPlatform:-linux/amd64}
     ports:
       - "5601:5601"
     volumes:
       - ./configurations/kibana.config.yml:/usr/share/kibana/config/kibana.yml
-  package-registry:
```

Comment:
If we get to an agreement to test PR versions of the package registry, then we can re-add this block again.